File: TOMIMSV4/Entity/Transacciones/Tarea_HandHeld/clsBeTarea_hh.vb
Public Class clsBeTarea_hh
    Implements ICloneable
    Implements IDisposable

    Public Property IdTareahh() As Integer = 0
    Public Property IdPropietario() As Integer = 0
    Public Property IdBodega() As Integer = 0
    Public Property IdMuelle() As Integer = 0
    Public Property IdEstado() As Integer = 0
    Public Property IdPrioridad() As Integer = 0
    Public Property IdTipoTarea() As Integer = 0
    Public Property IdTransaccion() As Integer = 0
    Public Property Tipo() As Integer = 0
    Public Property FechaInicio() As Date = Date.Now
    Public Property FechaFin() As Date = Date.Now
    Public Property DiaCompleto() As Boolean = False
    Public Property Asunto() As String = ""
    Public Property Ubicacion() As String = ""
    Public Property Descripcion() As String = ""
    Public Property Recordatorio() As String = ""
    Public Property IdOperadorBodega_Cerro As Integer = 0
    Public Property Host_Cerro As String = ""

    Sub New()
    End Sub

    Sub New(ByRef IdTareahh As Integer, ByVal IdPropietario As Integer, ByVal IdBodega As Integer, ByVal IdMuelle As Integer, ByVal IdEstado As Integer, ByVal IdPrioridad As Integer, ByVal IdTipoTarea As Integer, ByVal IdTransaccion As Integer, ByVal Tipo As Integer, ByVal FechaInicio As Date, ByVal FechaFin As Date, ByVal DiaCompleto As Boolean, ByVal Asunto As String, ByVal Ubicacion As String, ByVal Descripcion As String, ByVal Recordatorio As String)
        Me.IdTareahh = IdTareahh
        Me.IdPropietario = IdPropietario
        Me.IdBodega = IdBodega
        Me.IdMuelle = IdMuelle
        Me.IdEstado = IdEstado
        Me.IdPrioridad = IdPrioridad
        Me.IdTipoTarea = IdTipoTarea
        Me.IdTransaccion = IdTransaccion
        Me.Tipo = Tipo
        Me.FechaInicio = FechaInicio
        Me.FechaFin = FechaFin
        Me.DiaCompleto = DiaCompleto
        Me.Asunto = Asunto
        Me.Ubicacion = Ubicacion
        Me.Descripcion = Descripcion
        Me.Recordatorio = Recordatorio
    End Sub

    Public Function Clone() As Object Implements System.ICloneable.Clone
        Return MyBase.MemberwiseClone()
    End Function

End Class

File: WMS.EntityCore/Datos_Maestros/clsBeTarea_hh.cs
using System;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations.Schema;

namespace WMSWebAPI.Be
{
    public class clsBeTarea_hh : ICloneable
    {
        [Column("IdTareahh")]
        [DisplayName("IdTareahh")]
        public int IdTareahh { get; set; } = 0;

        [Column("IdPropietario")]
        [DisplayName("IdPropietario")]
        public int IdPropietario { get; set; } = 0;

        [Column("IdBodega")]
        [DisplayName("IdBodega")]
        public int IdBodega { get; set; } = 0;

        [Column("IdMuelle")]
        [DisplayName("IdMuelle")]
        public int IdMuelle { get; set; } = 0;

        [Column("IdEstado")]
        [DisplayName("IdEstado")]
        public int IdEstado { get; set; } = 0;

        [Column("IdPrioridad")]
        [DisplayName("IdPrioridad")]
        public int IdPrioridad { get; set; } = 0;

        [Column("IdTipoTarea")]
        [DisplayName("IdTipoTarea")]
        public int IdTipoTarea { get; set; } = 0;

        [Column("IdTransaccion")]
        [DisplayName("IdTransaccion")]
        public int IdTransaccion { get; set; } = 0;

        [Column("Tipo")]
        [DisplayName("Tipo")]
        public int Tipo { get; set; } = 0;

        [Column("FechaInicio")]
        [DisplayName("FechaInicio")]
        public DateTime FechaInicio { get; set; } = DateTime.Now;

        [Column("FechaFin")]
        [DisplayName("FechaFin")]
        public DateTime FechaFin { get; set; } = DateTime.Now;

        [Column("DiaCompleto")]
        [DisplayName("DiaCompleto")]
        public bool DiaCompleto { get; set; } = false;

        [Column("Asunto")]
        [DisplayName("Asunto")]
        public string Asunto { get; set; } = "";

        [Column("Ubicacion")]
        [DisplayName("Ubicacion")]
        public string Ubicacion { get; set; } = "";

        [Column("Descripcion")]
        [DisplayName("Descripcion")]
        public string Descripcion { get; set; } = "";

        [Column("Recordatorio")]
        [DisplayName("Recordatorio")]
        public string Recordatorio { get; set; } = "";

        [Column("IdOperadorBodega_Cerro")]
        [DisplayName("IdOperadorBodega_Cerro")]
        public int IdOperadorBodega_Cerro { get; set; } = 0;

        [Column("Host_Cerro")]
        [DisplayName("Host_Cerro")]
        public string Host_Cerro { get; set; } = "";
        public bool CreaTarea { get; set; }=false;        
        public bool IsNew { get; set; } = false;
        public clsBeTarea_hh() { }

        public object Clone()
        {
            return this.MemberwiseClone();
        }
    }
}
